frontend/src/util/validators.js

```diff
@@ -91,6 +91,47 @@ export function validateDOB(dateOfBirth) {
   return true
 }
 
+/**
+ * Validates that the Date is not earlier than '1900-01-01' .
+ */
+export function validateMinimumDate(dateInput) {
+  if (!helpers.req(dateInput)) {
+    return true
+  }
+  if (!dayjs(dateInput).isAfter(dayjs('1900-01-01'))) {
+    return false
+  }
+  return true
+}
+
+/**
+ * Validates that the EffectiveDate is not earlier than '1900-01-01' .
+ */
+export function validateMinimumEffectiveDate(dateInput) {
+  if (!helpers.req(dateInput)) {
+    return true
+  }
+  dateInput = new Date(this.coverageEffectiveDate.year, this.coverageEffectiveDate.month, 1)
+  if (!dayjs(dateInput).isAfter(dayjs('1900-01-01'))) {
+    return false
+  }
+  return true
+}
+
+/**
+ * Validates that the Cancel Date is not earlier than '1900-01-01' .
+ */
+export function validateMinimumCancelDate(dateInput) {
```

Comment:
Currently this method is equivalent to validateMinimumEffectiveDate. Please see my comment below.